verl/trainer/config/gsm8k_tool_config.yaml

```diff
@@ -0,0 +1,15 @@
+tools:
+  - class_name: "tool.gsm8k_tool.Gsm8kTool"
+    config:
```

Comment:
Here seems a little confused concept of `config` and `tool_schema`. `config` is used for config tool behavior, `tool_schema` is used for model chat template and function call parser. Here `config` content should be tool_schema